### ISSUE TITLE
@zurda (Michal Weizman)

### ISSUE THREAD

**Issue by arrested-developer** [fac-15/user-manuals#5]
# @zurda or Michal Weizman
## How I get my best work done: 
- 📚  I like having a little time to research things and for trial and error before starting to work on something completely new.  
- 🚜  I like building things gradually, for example building a program step by step and testing it does what's expected before moving on. 
- ❓ I'm really interested in the `why`. For example, if something doesn't work as I expected I rather search `why` before looking for a different solution. But don't let the fact that I really like digging into things slow the team down! I can do it in my spare time, so please lmk if it's not a priority.

## The role I usually take in a team:  
It really depends on the team.
-  ✅  I usually find it easier to follow a todo list of prioritised things, and having things organised in one place. If I see a task wasn't picked up by anyone - I'd be eager to pick it up and get it done (after letting everyone know I'm about to do it). 
 - 🦄 I also enjoy the planning phase since it's the best time to be creative and think out of the box 

## My communication style: 
- 🗣 I sometimes need a little more time to phrase things the right way, so I apologize in advance. If you feel you know where I'm going with something, and it takes me ages to say it - please feel free to lmk :) 

## What I value:
Honesty. Efficiency. Creativity. 
- 🤔 I value taking time to make things more efficient. It's up to the team, of course, but for personal projects, I rather have many efficiently built functions than having the project done in a non-DRY way.

## What people misunderstand about me: 
- I'm might be the last to share my preference in a group discussion. I would much rather take the group's/ majority route than speak out. I guess the reason is me being a bit intimidated by confrontation - but I am always happy to share my thoughts. The thing is you might need to ask me directly for my preference... (sorry about that - I'm working on it!).
- 😨  I sometimes get overwhelmed among large groups of people I don't know very well, so if I get quiet it's probably my social anxiety rather than lack of interest. 
- 🇮🇱 ▶️ 🇬🇧 A lot of times I'm afraid I might have used the wrong word or misunderstood a social etiquette, so honesty is extremely appreciated. I'd appreciate any feedback 

## How I like to get my feedback: 
- I really appreciate honesty. For me, being here is more than learning how to become a better developer. It's also about working in teams, so I welcome both professional and personal feedback. Please feel free to lmk. Weather you choose to do it when the entire team is around, or in person - I honestly want to know and improve ❤️
